src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.appointment;
+
+import java.util.UUID;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represents an Appointment in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Appointment {
```

Comment:
Should add a method: `boolean overlap(Appointment)` to compare appointment for overlaps.